IBPSA/Fluid/Movers/FlowControlled_m_flow.mo

```diff
@@ -20,7 +20,22 @@ model FlowControlled_m_flow
           IBPSA.Fluid.Movers.BaseClasses.Characteristics.flowParameters(
             V_flow = {i/(nOri-1)*2.0*m_flow_nominal/rho_default for i in 0:(nOri-1)},
             dp =     {i/(nOri-1)*2.0*dp_nominal for i in (nOri-1):-1:0}),
-      final use_powerCharacteristic = if per.havePressureCurve then per.use_powerCharacteristic else false),
+        final etaHydMet=
```

Comment:
These changes look like a duplicate of the changes in `IBPSA/Fluid/Movers/FlowControlled_dp.mo`. Is it possible to avoid this using the existing partials?

Reply:
This is something I have thought about. There is a lot of code shared by the two flow-controlled models (`FlowControlled_dp` and `FlowControlled_m_flow`) and the two speed-controlled models (`SpeedControlled_Nrpm` and `SpeedControlled_y`). All four models extend `BaseClasses.PartialFlowMachine`.
These pairs share top-level declarations. These top-level declarations are awkward to be moved down to `BaseClasses.PartialFlowMachine` because they are not used by the other pair.
I would suggest creating a new layer of partial model in between them, say `BaseClasses.PartialFlowControlled` and `BaseClasses.PartialSpeedControlled`. This should be helpful tidying up all the copy-pastas. The downside is that this makes the `Movers` package even more convoluted and difficult to read and maintain. (But as a former C# user, I have always thought we haven't abused the inheritance nearly enough in Buildings and IBPSA libraries.)
I can work on this if others agree this is helpful.